server/server/src/middleware/mod.rs

```diff
@@ -1,12 +1,17 @@
-pub mod content_length_limit;
+use log::Level;
+
 mod central_server_only;
+pub mod content_length_limit;
 
 pub fn compress() -> actix_web::middleware::Compress {
     actix_web::middleware::Compress::default()
 }
 
 pub fn logger() -> actix_web::middleware::Logger {
-    actix_web::middleware::Logger::default()
+    actix_web::middleware::Logger::new(r#""%r" FROM: "%{Referer}i" STATUS: %s"#)
+        .log_level(Level::Info)
+        .log_target("request_log")
+        .exclude("/graphql")
```

Comment:
excluding completely... maybe we still want to show the request metadata though? like you say, might become clear as we work more on logging, get clearer on requirements...